app/workers/purge_test_applications.rb

```diff
@@ -0,0 +1,29 @@
+class DeleteTestApplications
+  include Sidekiq::Worker
+
+  def perform(*)
+    raise 'You can only purge test applications in a test environment' unless DeleteTestApplications.can_run_in_this_environment?
+
+    candidates_to_purge.find_each do |candidate|
+      candidate.application_forms.each do |application_form|
+        application_form.application_choices.each(&:destroy)
+        application_form.destroy
+      end
+      candidate.destroy
+    end
```

Comment:
Because we've set up [cascading deletes correctly](https://github.com/DFE-Digital/apply-for-teacher-training/blob/65491ad206b25723058eb38a31da737bc1776e51/spec/models/candidate_spec.rb#L15-L35), we should be able to do this:

```suggestion
    candidates_to_purge.delete_all
```